gluonnlp/data/candidate_sampler.py

```diff
@@ -34,61 +31,3 @@ class CandidateSampler(object):
 
     def __call__(self):
         raise NotImplementedError
-
-
-class UnigramCandidateSampler(CandidateSampler):
```

Comment:
why is it moved to scripts? It should be quite commonly used

Reply:
Due to the inconsistency with sharing samples across batches with the sampler for language models. I moved it back but keep the accidental hit masking in scripts meaning to avoid any inconsistency.